src/RequestExecutor/CertificatesLoader.ts

```diff
@@ -0,0 +1,37 @@
+import loadWinCertificates from 'win-ca';
+import https from 'https';
+import fs from 'fs';
+import { promisify } from 'util';
+
+const readFile = promisify(fs.readFile);
+
+export class CertificatesLoader {
+  async load(): Promise<void> {
+    const os = process.platform;
+    switch (os) {
+      case 'win32':
+        loadWinCertificates();
+        break;
+      default:
+        await this.loadLinuxCertificates();
+        break;
```

Comment:
`default` branch should do nothing, otherwise, it will lead to unexpected behavior in case of BSD or other different OS